backend/src/common/util/workflow.go

```diff
@@ -51,11 +51,17 @@ func (w *Workflow) OverrideParameters(desiredParams map[string]string) {
 		if param, ok := desiredParams[currentParam.Name]; ok {
 			desiredValue = &param
 		} else {
-			desiredValue = currentParam.Value
+			val := currentParam.Value.String()
+			desiredValue = &val
+		}
+		val := ""
+		if desiredValue != nil {
+			val = *desiredValue
 		}
 		desiredSlice = append(desiredSlice, workflowapi.Parameter{
+
 			Name:  currentParam.Name,
-			Value: desiredValue,
+			Value: workflowapi.AnyStringPtr(val),
```

Comment:
When `val` is nil, is there a problem assigning it as empty string? (seems like the case here)

Reply:
Good point, I think we should go with `nil` instead, will update.